code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -263,16 +263,17 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/chair/stool, 0)
 /obj/structure/chair/stool/narsie_act()
 	return
 
-/obj/structure/chair/mouse_drop_dragged(atom/over_object, mob/user, src_location, over_location, params)
-	if(over_object == user)
-		if(!item_chair || has_buckled_mobs())
-			return
-		user.visible_message(span_notice("[user] grabs \the [src.name]."), span_notice("You grab \the [src.name]."))
-		var/obj/item/C = new item_chair(loc)
-		C.set_custom_materials(custom_materials)
-		TransferComponents(C)
-		user.put_in_hands(C)
-		qdel(src)
+/obj/structure/chair/mouse_drop_dragged(atom/over_object, mob/living/user, src_location, over_location, params)
+	if(!istype(user) || over_object != user)
+		return
```

Comment:
Types should be what something IS rather then what we want it to be. if you want to check for living be explicit rather then implicit about it. (for the sake of clarity)
```suggestion
/obj/structure/chair/mouse_drop_dragged(atom/over_object, mob/user, src_location, over_location, params)
	if(!isliving(user) || over_object != user)
		return
```